src/http/verification.rs

```diff
@@ -0,0 +1,39 @@
+use html_parser::Dom;
+use std::str::FromStr;
+use url::Url;
+
+pub async fn is_verified(input_url: Url, expected_url: Url) -> bool {
+    let website = reqwest::get(input_url.to_string()).await;
```

Comment:
I believe that to be verified, the target has to be `https` as well

Reply:
I don't think that's the case, at least I couldn't find any documentation/wiki that said only HTTPS sites can be verified.

Reply:
I thought I read it somewhere, and found it in the [Mastodon docs on verification](https://docs.joinmastodon.org/user/profile/#verification). I suspect the reason is that if the link isn't HTTPS, you could be tricked into verifying through a man-/person-in-the-middle attack.

Reply:
You're right, I'll update the implementation to return false on non-HTTPS URLs :)
Attackers won't have an good time with this :smiling_imp: